draft-ietf-ccamp-rfc9093-bis.md

```diff
@@ -264,180 +264,145 @@ Please replace XXXX with the RFC number assigned to this document and remove thi
+generic topology model defines the label-hop in {{!RFC8795}}.
+: This grouping shows the WSON-specific label-hop information. See {{label-range}} for more details.
+
+flexi-grid-label-range-info:
+: A YANG grouping that defines flexi-grid label range information as
+defined in {{!RFC8363}}. See {{label-range}} for more details. See {{label-range}} for more details.
+
+flexi-grid-label-step:
+: A YANG grouping that defines flexi-grid label steps as defined in
+{{!I-D.ietf-teas-rfc8776-update}}. See {{label-range}} for more details.
+
+wdm-label-start-end:
+: A YANG grouping that combines the definition of label-start/label-end information
+that was defined separately in wson-label-start-end and flexi-grid-label-start-end,
+to support optical network scenarios that contain both fixed- and flexi-grid
+links. See {{label-range}} for more details.
+
+wdm-label-hop:
+: A YANG grouping that combines the definition of label hop information
+that was defined separately in wson-label-hop and flexi-grid-label-hop,
+to support optical network scenarios that contain both fixed- and flexi-grid
+links. See {{label-range}} for more details.
+
+wdm-label-range-info:
+: A YANG grouping that combines the definition of label range information
+that was defined separately in wson-label-range-info and flexi-grid-label-range-info,
+to support optical network scenarios that contain both fixed- and flexi-grid
+links. See {{label-range}} for more details.
+
+wdm-label-step:
+: A YANG grouping that combines the definition of label step information
+defined separately in wson-label-step and flexi-grid-label-step,
+to support optical network scenarios that contain both fixed- and flexi-grid
+links. See {{label-range}} for more details.
 
 transceiver-capabilities:
-
-> A YANG grouping to define the transceiver capabilities (also called
+: A YANG grouping to define the transceiver capabilities (also called
 "modes") needed to determine optical signal compatibility.
 
 standard-mode:
-
-> A YANG grouping for the standard modes defined in {{ITU-T_G.698.2}}.
+: A YANG grouping for the standard modes defined in {{ITU-T_G.698.2}}.
 
 organizational-mode:
-
-> A YANG grouping to define transponder operational mode supported by
+: A YANG grouping to define transponder operational mode supported by
 organizations or vendors, as defined in {{!I-D.ietf-ccamp-optical-impairment-topology-yang}}.
 
-common-explicit-mode:
-
-> A YANG grouping to define the list of attributes related to optical
+explicit-mode:
+: A YANG grouping to define the list of attributes related to optical
 impairments limits in case of transceiver explicit mode, as defined in {{!I-D.ietf-ccamp-optical-impairment-topology-yang}}.
+: Note that the the actual portion of the spectrum occupied by an OTSi is not explicitly reported within the explicit-mode parameters because it can calculated using the available-baud-rate, the roll-off and the min-carrier-spacing attributes.
```

Comment:
Minor comment: may be better to write ".... it can **be** calculated using the ...."

Reply:
thanks! ok

Reply:
Fixed with 844527b1a0d069cb34558bf0565acf9840fa1c4f